config/elastic_apm.yml

```diff
@@ -1,3 +1,4 @@
+enabled: <%= ENV["ELASTIC_APM_ENABLED"] == 'true'%>
```

Comment:
Yes. Nice. According to the elastic java docs (first one I could find):

> Setting to false will completely disable the agent, including instrumentation and remote config polling. If you want to dynamically change the status of the agent, use [recording](https://www.elastic.co/guide/en/apm/agent/java/master/config-core.html#config-recording) instead.

Have you also tested whether the system can now run without having the other environment variables set? If yes, then I approve this PR.

Reply:
I ran it locally with RAILS_ENV set to production and got no errors.